dvc/utils/__init__.py

```diff
@@ -382,3 +382,8 @@ def relpath(path, start=os.curdir):
     ):
         return path
     return os.path.relpath(path, start)
+
+
+def chunk(iterable, chunk_size):
```

Comment:
Ok, how about making `to_chunks` accept both desired number of chunks or chunk size? I really don't like having both `to_chunks` and `chunk` at the same time.

Reply:
I can work with that :)